README.md

```diff
@@ -11,6 +11,18 @@ Welcome to Braintree's Drop-in SDK for iOS!
 
 **The Braintree iOS Drop-in SDK permits a deployment target of iOS 12.0 or higher.** It requires Xcode 15.0+ and Swift 5.9+.
 
+## 📣 Announcements
```

Comment:
Would it make sense to move the announcement to top of README?

Reply:
+1, we should put this up in lights as clearly as possible

Reply:
Maybe even some scary-looking alarm emojis, or things to grab people's eye.

What about also updating the repo description with some warning message?

Reply:
I was thinking about that since someone needs to scroll past the image to see the announcement